### PR TITLE
fix(scalyr): #153 - sanitize error token

### PR DESCRIPTION
**Pull request by ahmed-shehata**
fix(scalyr): #153 - sanitize error token

### CODE REVIEW

**Comment by ahmed-shehata** [comment]
👍

File: src/api/atlas_sdk_client.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ function fetchEndpoint(endpoint) {
     .then(response => {
       return endpoint.transform(response);
     }).catch(error => {
+      error.response.headers.set('Authorization', 'BEARER XXX');
```
Comment (Etchs) [inline src/api/atlas_sdk_client.js]: maybe safer `error.response && error.response.headers.set('Authorization', 'BEARER XXX');`?

**Comment by ahmed-shehata** [comment]
👍

**Comment by Etchs** [comment]
👍

**Pull request merged**